SU-SWS/saa_victoria_subtheme
# Adapt 738  cta line height

# READY FOR REVIEW

# Summary
- Change line height of cta links
- Right now cta links are inheriting the line height 1.4 from paragraph tags, and instead use the correct 1.25 line height.
- Note that regular links will need to adopt the paragraph line height as they can be used in or outside of a paragraph..

# Review By (Date)
- 9/23

# Urgency
- low

# Steps to Test

1. Pull down the branch or see https://saauserguide-test.sites.stanford.edu/site-building-guide/saa-components-stanford-sites/text-area
2. Clear the site cache
3. Check that all links with any cta classes (su-link--action, su-link--external, etc) have line-height of 1.25
4. Test links within an icon card here: https://saauserguide-test.sites.stanford.edu/site-building-guide/saa-components-stanford-sites/cards/icon-cards
5. Test link within in-page banners here: https://saauserguide-test.sites.stanford.edu/site-building-guide/saa-components-stanford-sites/page-banners

# Affected Projects or Products
- N/A

# Associated Issues and/or People
- JIRA ticket ADAPT-738
